test/integration/stats/data/service_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Stats::Data::ServicesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
What's this test doing and why should it fail?

Reply:
What do you mean with "why should it fail"? I don't understand the question.

Reply:
@josemigallas It's testing basic requests to the `top_application` endpoint. Right now it fails with status code 400

Reply:
I wonder what's the relation between the change in the controller and this test. If the test is right now failing with a 400, then we should also add a test case for that. The test must prove that the new changes are useful.

Reply:
This test case covers it, we were missing a basic test case that would reveal the bug.